File: official/docs/golang/current/rates/regenerate.go
package example

import (
	"fmt"

	"github.com/EasyPost/easypost-go/v5"
)

func regenerate() {
	client := easypost.New("EASYPOST_API_KEY")

	shipment, _ := client.RerateShipment("shp_...")

	fmt.Println(shipment)
}

File: official/docs/java/v7/rates/regenerate.java
package shipments;

import com.easypost.exception.EasyPostException;
import com.easypost.model.Shipment;
import com.easypost.service.EasyPostClient;

public class Regenerate {
    public static void main(String[] args) throws EasyPostException {
        EasyPostClient client = new EasyPostClient("EASYPOST_API_KEY");

        Shipment shipment = client.shipment.newRates("shp_...");

        System.out.println(shipment);
    }
}
